examples/node/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "test-node",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": ""
```

Lets re-use the same information than the root package
```suggestion
  "name": "VASTClient-node-example",
  "version": "1.0.0",
  "main": "index.js",
  "type": "module",
  "scripts": {
    "start": "node index.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "Dailymotion <vast-client@dailymotion.com> (https://developer.dailymotion.com/)"
  "license": "MIT",
  "description": "VAST Client node usage example"
```